api-testing/multicloud-api/src/main/java/com/soda/utils/Utils.java

```diff
@@ -0,0 +1,131 @@
+/*
+  Copyright 2020 The SODA Authors.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.soda.utils;
+
+import java.io.File;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+/**
+ * This class used for common utils.
+ */
+public class Utils {
+
+    /**
+     * format strings for the date/time and date stamps required during signing
+     **/
+    public static final String ISO8601BasicFormat = "yyyyMMdd'T'HHmmss'Z'";
+
+    /**
+     * Get List of files
+     *
+     * @param beginPattern begin pattern (bucket) e.g: bucket_b237
+     * @param path folder path e.g: resource/inputs
+     * @return list of files
+     */
+    public static List<File> listFilesMatchingBeginsWithPatternInPath(final String beginPattern, String path) {
```

Comment:
Method name is too long

Reply:
I have pushed the changes as per your review comments. Please review.